### DISCUSSION TITLE
<title>Requesting help for finding an option in communicate using markdown module <\title>

### DISCUSSION THREAD

**Discussion by Sahithiaele** [community/community#113698] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/134089299/8bbf583d-4fae-4286-97dd-3b939bf6c2d2)
unable to find the files changed in the pull request in step one while going through the communicate using markdown module.

**Comment by nagasriramnani** [comment]
In the image you've provided, there are instructions for editing a file with headers using Markdown in the context of a GitHub pull request. If you're following these instructions but cannot find the "Files changed" tab mentioned in step 5, this could be due to a few reasons:

The Pull Request Doesn't Contain Changes: If the pull request was created without any changes, or if the changes were not committed properly, there would be no "Files changed" tab to select.

The Pull Request Page Hasn't Loaded Correctly: Sometimes web pages don't load fully due to network issues or browser glitches. Refreshing the page can sometimes resolve this.

The User Interface Has Changed: If the GitHub user interface has been updated since these instructions were written, the "Files changed" tab might have been moved or renamed.

The Wrong Pull Request: Ensure that you're looking at the correct pull request that you intended to view or edit.

Here are some steps to troubleshoot the issue:

Make sure you are on the correct pull request page. You should see tabs like "Conversation," "Commits," "Checks," "Files changed," etc., near the top of the pull request page.
If you don't see any changes, you might want to check the commit history to ensure that the file index.md was indeed changed and committed.
If you can't find the file, you may need to go back to the repository's main page, find the branch you're working on, and ensure that the changes have been committed there.
In case the instructions are out of date with the current GitHub interface, look for a tab or section that suggests reviewing changes, which might be named differently.
If you're still unable to find the "Files changed" tab or equivalent, consider looking for help on the GitHub documentation or reaching out to GitHub support for assistance with their interface.
If none of these steps help, please provide additional details about the issue, such as whether you were able to create the pull request successfully, whether you can see other tabs like "Conversation" and "Commits," and any other relevant information that could assist in troubleshooting.
